common/scala/src/main/scala/whisk/core/containerpool/HttpUtils.scala

```diff
@@ -126,9 +126,8 @@ protected[core] class HttpUtils(hostname: String, timeout: FiniteDuration, maxRe
     } match {
       case Success(response) => response
       case Failure(t: RetryableConnectionError) if retry =>
-        val sleepTime = 10.milliseconds
+        val sleepTime = 50.milliseconds
```

Comment:
@sven-lange-last Can this hardcoded value value be be a configuration setting that can be overload thru prureconfig/environment-variable?

It was originally to retry every 100ms, then it was changed to 10ms, and now every 50ms, 
In case 50ms also makes the invokers slower overall and need to be adjusted again, doing on config is more manageable.

Reply:
@csantanapr Let me put it that way: it may be tempting to make this value configurable. I think that this setting should not be tampered with to save an invoker. This value is used in a `Thread.sleep()` which occupies the current worker thread. Increasing the value may have pretty undesired effects. We should find a reasonable value and don't change it any more.

It would be much more important to replace the `Thread.sleep()` with something that frees the thread while waiting / sleeping.